CHANGELOG.md

```diff
@@ -4,7 +4,11 @@
 
 - Components
     - `icu_provider`
-        - `GeneralCategoryGroup::contains` now accepts `self` by value (unicode-org#5952) 
+        - `GeneralCategoryGroup::contains` now accepts `self` by value (unicode-org#5952)
+    - `icu_locale`
+        - `LocaleExpander`, `LocaleDirectionality`, and `LocaleCanonicalizer` distinguish between `new_common()` and `new_extended()` constructors (unicode-org#5958)
+    - `icu_segmenter`
+        - Segmenters that can take a content locale now specify `_root()` on their default localeless constructors (unicode-org#5958)
```

Comment:
"root" is a CLDR term that we have so far avoided. I don't think it's very clear if you don't think of languages as a tree.

Reply:
Happy to bikeshed the names further.

https://github.com/unicode-org/icu4x/issues/5856#issuecomment-2505028516 has some discussion. I don't really like any of the names on the table.

Reply:
I dislike root for the same reason, it is CLDR jargon, however it is a relatively convenient name.